assets/pages/bundles-landing/components/Bundles.jsx

```diff
@@ -256,6 +290,20 @@ function ContributionBundle(props: PropTypes) {
         onNumberInputKeyPress={onClick}
         {...props}
       />
+
+      {props.contribType === 'ONE_OFF' && !!contribAttrs.paypalCta &&
+        <PayPalContributionButton
+          buttonText={contribAttrs.paypalCta.text}
+          amount={Number(props.contribAmount.oneOff.value)}
+          referrerAcquisitionData={props.referrerAcquisitionData}
+          isoCountry={props.isoCountry}
+          // eslint-disable-next-line no-alert
+          errorHandler={e => alert(e)}
+          abParticipations={props.abTests}
+          additionalClass={`bundle-landing ${props.contribError ? 'contrib-error' : ''}`}
+        />
```

Comment:
As discussed offline, in the case where a user enters an invalid value into the 'other amount' box, this button will still be clickable, and will send a bad request to the contributions backend.

To prevent this I think you could just pass `canClick={!props.contribError}` and the button will take care of the rest.